src/main/java/busTickets/busTicketStorage/BusTicketStorage.java

```diff
@@ -0,0 +1,67 @@
+package busTickets.busTicketStorage;
+
+import busTickets.BusTicket;
+import busTickets.BusTicketValidation;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BusTicketStorage {
+    public ArrayList<BusTicket> busTickets = new ArrayList<>();
+
+    public BusTicketStorage() {
+        try {
```

Comment:
in constructor we're not putting business logic -> we need to do it separately and put here list of BusTicket as a param

Reply:
I created another class to handle json file where method of returning type returns me array of busTickets